src/System.Device.Gpio/System/Device/I2c/Devices/I2cDevice.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Device.I2c
+namespace System.Device.I2c.Devices
```

Comment:
Is this change expected as well? I'd expect I2cDevice to live under System.Device.I2c and implementations under .Devices. cc: @joperezr

Reply:
I was thinking this was part of...

> 3. I2cDevice.cs moved to Devices folder.

Reply:
don't have strong opinion, it just feels weird from naming point of view. @joperezr, thoughts?

Reply:
mm I also thought about this, but I think it does make sense to have it moved to this new namespace as well. It makes sense that a Device is on the Devices namespace :)

Reply:
@joperezr I'm wondering, perhaps we should make Windows/LinuxI2cDevice internal (I don't see good justification for having them right now) and just don't rename namespace (just add directories)

Reply:
We talked offline with @joperezr and we will only have 1 I2cDevice class which collapses the logic of both Linux and Windows and I2cDevice.Create will become `new I2cDevice`. We are also thinking of doing similar thing for board drivers but that one has a bit more justification on having them separate so we need to figure out what should happen.

Reply:
We will do that in a separate PR though, so we can take this in first and we will make the required changes subsequently

Reply:
Do I need to create the next PR to update the bindings to reflect these changes or wait?  Also, I was going to create another PR similar to this for SPI.  Do I need to hold off on that base on the planned changes?